tests/tt_metal/tt_metal/perf_microbenchmark/9_dram_adjacent_read_remote_l1_write/kernels/reader_dram.cpp

```diff
@@ -0,0 +1,117 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
add a comment here that explains why this test writes to NOC registers directly and how it is different from other reader/writer tests

Reply:
added the comments at the host code side